core/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/InternalTerms.java

```diff
@@ -112,6 +113,10 @@ public long getDocCountError() {
             return docCountError;
         }
 
+        public long getBucketOrd() {
```

Comment:
Removed this method and refactored the code here: https://github.com/elastic/elasticsearch/pull/22343/commits/8ddaf23f9d181015e6e38941c786c78500a5159c